### DISCUSSION TITLE
Test refactored popup

### DISCUSSION THREAD

**Discussion by stabldev** [skeletonlabs/skeleton#1425] [Q&A] [answered]
Hey! I saw that PR with fixing many popup related issues. And I like to test it out.
I tried this command to install from `popup-rework` branch:
```bash
npm i -D https://github.com/skeletonlabs/skeleton.git#feat/popup-rework
```

But its not working, and in `devDependencies` it shows like this
```bash
"@skeletonlabs/skeleton": "github:skeletonlabs/skeleton#feat/popup-rework",
```
uhh! is there any way to test it?

**Comment by Sarenor** [comment, marked as answer]
https://github.com/skeletonlabs/skeleton/wiki/Create-and-Install-Test-Packages#install-a-tarball-package
There's a guide to build your own tarball package and how to use it
